sites/main-site/src/content/special-interest-groups/animal-genomics/index.md

```diff
@@ -29,4 +29,8 @@ Finally, the group will be a place where users can share their experience with n
 
 # Activities
 
-The format will involve a monthly meeting featuring a presentation and round table discussions on pre-identified topics. The presentation will typically feature novel ongoing efforts in animal genomics (new consortiums, new calls, etc). The discussion will often be a follow-up on the talk.
+The format will involve a [monthly meeting](meetings.md) featuring a presentation and round table discussions on pre-identified topics. The presentation will typically feature novel ongoing efforts in animal genomics (new consortiums, new calls, etc). The discussion will often be a follow-up on the talk.
+
+# How to join
+
+Join the [nf-core Slack](/join#slack) and find us in the [#animal-genomics channel](https://nfcore.slack.com/archives/CE5EL6326)
```

Comment:
The link does not take me to the #animal-genomics channel thus, I will delete it
```suggestion
Join the [nf-core Slack](/join#slack) and find us in the #animal-genomics channel
```